src/EnergyPlus/SystemAvailabilityManager.cc

```diff
@@ -792,7 +792,7 @@ namespace SystemAvailabilityManager {
 				SetupOutputVariable( "Availability Manager Optimum Start Control Status []", OptStartSysAvailMgrData( SysAvailNum ).AvailStatus, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name );
 
 				// add
-				SetupOutputVariable( "Availability Manager Optimum Start Hours Before Occupancy []", OptStartSysAvailMgrData( SysAvailNum ).NumHoursBeforeOccupancy, "System", "Average", OptStartSysAvailMgrData( SysAvailNum ).Name, "Daily" );
```

Comment:
Agreed.  @mjwitte if you'd like me to get these changes into the bug fix release, I'm happy to merge it without this change.  I can also revert out that change and push it up if you are unable.